pkg/ddevapp/settings_test.go

```diff
@@ -82,3 +93,174 @@ func TestWriteDrushConfig(t *testing.T) {
 	err = os.RemoveAll(dir)
 	assert.NoError(t, err)
 }
+
+// TestIncludeSettingsDdevInNewSettingsFile verifies that when no settings.php file exists,
+// a settings.php file is created that includes settings.ddev.php.
+func TestIncludeSettingsDdevInNewSettingsFile(t *testing.T) {
+	dir := testcommon.CreateTmpDir("")
```

Comment:
IMO it's a little better to give a prefix here, like `dir := testcommon.CreateTmpDir("TestIncludeSettingsDdevInNewSettingsFile")`; it doesn't affect functionality or improve the test, but makes debugging (and possibly cleanup) easier and more obvious.